_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -1521,27 +1464,27 @@ mfxStatus VideoDECODEVP9_HW::DecodeFrameHeader(mfxBitstream *in, VP9DecoderFrame
     }
     catch(const vp9_exception & ex)
     {
-        return ConvertStatusUmc2Mfx(ex.GetStatus() );
+        mfxStatus sts = ConvertStatusUmc2Mfx(ex.GetStatus());
+        MFX_CHECK_STS(sts);
+        return sts;
     }
 
     return MFX_ERR_NONE;
 }
 
 mfxStatus VideoDECODEVP9_HW::PackHeaders(mfxBitstream *bs, VP9DecoderFrame const & info)
 {
-    if ((NULL == bs) || (NULL == bs->Data))
-        return MFX_ERR_NULL_PTR;
+    MFX_CHECK_NULL_PTR1(bs);
+    MFX_CHECK_NULL_PTR1(bs->Data);
 
     if (!m_Packer.get())
```

Comment:
```.get()``` call is unnecessary, all smart pointers have ```bool operator()```, so ```if (!m_Packer)``` will also works as if it was regular pointer